cartographer/io/image_utils.cc

```diff
@@ -0,0 +1,67 @@
+#include "cartographer/io/image_utils.h"
```

Comment:
I do not agree with *utils.h name of this file. How about just calling it Image?